src/test/rpc/AccountSet_test.cpp

```diff
@@ -41,7 +41,7 @@ class AccountSet_test : public beast::unit_test::suite
         env.fund(XRP(10000), noripple(alice));
         // ask for the ledger entry - account root, to check its flags
         auto const jrr = env.le(alice);
-        BEAST_EXPECT((*env.le(alice))[sfFlags] == 0u);
+        BEAST_EXPECT(jrr && jrr->at(sfFlags) == 0u);
```

Comment:
In this file, there are 22 other dereferences of the `*env.le()` function. Visually, I'm convinced that none of them are null-pointers, because all the `Account` objects have been funded with some XRP.

But is it necessary to perform this type of null-pointer-check on all of them?

Reply:
It wouldn't hurt to check. Even though it's been a while, I think it was the double call to `env.le()` that caught my attention, and then I just checked for null since I was changing it anyway.

Reply:
I realize that I did note "Maybe there are more of these?" in the "Future Tasks" section of the description.